### ISSUE TITLE
template indent is not correct after created

### ISSUE THREAD

**Issue by dsx1123** [CiscoDevNet/ansible-dcnm#62]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Ansible Version and collection version

```
ansible 2.10.5
  config file = /root/.ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /root/virtualenv/ansible/lib/python3.7/site-packages/ansible
  executable location = /root/virtualenv/ansible/bin/ansible
  python version = 3.7.3 (default, Apr  3 2019, 05:39:12) [GCC 8.3.0]
```

### DCNM version

* V 11.5.1

### Affected module(s)

* dcnm_template

### Ansible Playbook

```yaml
# Copy-paste your anisble playbook here 
---
- name: create template for telemery
  hosts: dcnm
  gather_facts: false
  tasks:
    - name: read template config from file
      set_fact:
        template_content: "{{ lookup('file', './telemetry.cfg')}}"

    - name: create template
      cisco.dcnm.dcnm_template:
        config:
          - name: telegraf_telemetry_configuration
            description: "telemetry configuration"
            content: "{{ template_content}}"
```
content of  telemetry.cfg
```
feature telemetry

telemetry
  certificate /bootflash/telegraf.crt telegraf
  destination-profile
    use-vrf management
  destination-group 1
    ip address 10.195.225.176 port 57000 protocol gRPC encoding GPB
  sensor-group 1
    data-source DME
    path sys/ch depth unbounded
  sensor-group 2
    data-source DME
    path sys/intf depth unbounded
  sensor-group 3
    data-source DME
    path sys/bgp depth unbounded
  sensor-group 4
    data-source DME
    path sys/procsys/syscpusummary/syscpuhistory-last60seconds
  sensor-group 5
    data-source DME
    path sys/procsys/sysmem/sysmemusage
  sensor-group 6
    data-source DME
    path sys/bd depth unbounded
  sensor-group 7
    data-source DME
    path sys/mac depth unbounded
  sensor-group 8
    data-source DME
    path sys/evpn depth 4
  sensor-group 9
    data-source DME
    path sys/urib depth unbounded query-condition rsp-foreign-subtree=ephemeral
  sensor-group 10
    data-source DME
    path sys/u6rib depth unbounded query-condition rsp-foreign-subtree=ephemeral
  sensor-group 11
    data-source DME
    path sys/bgp/inst/dom-default/af-[l2vpn-evpn] depth unbounded query-condition rsp-foreign-subtree=ephemeral
  subscription 1
    dst-grp 1
    snsr-grp 1 sample-interval 10000
  subscription 2
    dst-grp 1
    snsr-grp 2 sample-interval 10000
  subscription 3
    dst-grp 1
    snsr-grp 3 sample-interval 30000
  subscription 4
    dst-grp 1
    snsr-grp 4 sample-interval 15000
  subscription 5
    dst-grp 1
    snsr-grp 5 sample-interval 15000
  subscription 6
    dst-grp 1
    snsr-grp 6 sample-interval 0
  subscription 7
    dst-grp 1
    snsr-grp 7 sample-interval 10000
  subscription 8
    dst-grp 1
    snsr-grp 8 sample-interval 15000
  subscription 9
    dst-grp 1
    snsr-grp 9 sample-interval 15000
  subscription 10
    dst-grp 1
    snsr-grp 10 sample-interval 15000
  subscription 11
    dst-grp 1
    snsr-grp 11 sample-interval 15000
```

### Debug Output

https://gist.github.com/dsx1123/6f39fc58304c69f8d88570781a9b5ad3

### Expected Behavior

the content of template should match the file

### Actual Behavior

Here is the actual content after create template:
```
feature telemetry

telemetry
 certificate /bootflash/telegraf.crt telegraf
 destination-profile
 use-vrf management
 destination-group 1
 ip address 10.195.225.176 port 57000 protocol gRPC encoding GPB
 sensor-group 1
 data-source DME
 path sys/ch depth unbounded
 sensor-group 2
 data-source DME
 path sys/intf depth unbounded
 sensor-group 3
 data-source DME
 path sys/bgp depth unbounded
 sensor-group 4
 data-source DME
 path sys/procsys/syscpusummary/syscpuhistory-last60seconds
 sensor-group 5
 data-source DME
 path sys/procsys/sysmem/sysmemusage
 sensor-group 6
 data-source DME
 path sys/bd depth unbounded
 sensor-group 7
 data-source DME
 path sys/mac depth unbounded
 sensor-group 8
 data-source DME
 path sys/evpn depth 4
 sensor-group 9
 data-source DME
 path sys/urib depth unbounded query-condition rsp-foreign-subtree=ephemeral
 sensor-group 10
 data-source DME
 path sys/u6rib depth unbounded query-condition rsp-foreign-subtree=ephemeral
 sensor-group 11
 data-source DME
 path sys/bgp/inst/dom-default/af-[l2vpn-evpn] depth unbounded query-condition rsp-foreign-subtree=ephemeral
 subscription 1
 dst-grp 1
 snsr-grp 1 sample-interval 10000
 subscription 2
 dst-grp 1
 snsr-grp 2 sample-interval 10000
 subscription 3
 dst-grp 1
 snsr-grp 3 sample-interval 30000
 subscription 4
 dst-grp 1
 snsr-grp 4 sample-interval 15000
 subscription 5
 dst-grp 1
 snsr-grp 5 sample-interval 15000
 subscription 6
 dst-grp 1
 snsr-grp 6 sample-interval 0
 subscription 7
 dst-grp 1
 snsr-grp 7 sample-interval 10000
 subscription 8
 dst-grp 1
 snsr-grp 8 sample-interval 15000
 subscription 9
 dst-grp 1
 snsr-grp 9 sample-interval 15000
 subscription 10
 dst-grp 1
 snsr-grp 10 sample-interval 15000
 subscription 11
 dst-grp 1
 snsr-grp 11 sample-interval 1500
```

### Steps to Reproduce

1. creat template use play above

**Comment by mikewiebe** [comment]
@dsx1123 Thanks for filing the issue.  We are currently working on a fix

**Comment by mmudigon** [comment]
The issue is fixed in #69
